server/bleep/src/env.rs

```diff
@@ -0,0 +1,69 @@
+use Feature::*;
+
+pub(crate) enum Feature {
+    /// Allow scanning any path on the system. This is dangerous!
+    AnyPathScan = 1 << 0,
```

Comment:
`1 << 0` defaults to `isize` by default, you can add `#[repr(u16)]` or similar to use an unsigned variant.